src/Uno.UI/UI/Xaml/Media/Animation/EasingColorKeyFrame.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Windows.UI.Xaml.Media.Animation
+{
+	partial class EasingColorKeyFrame : ColorKeyFrame
+	{
+		public static DependencyProperty EasingFunctionProperty = DependencyProperty.Register(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'EasingFunctionProperty' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)